experimental/generation/generator/packages/library/templates/standard/en-us/form-DisplayOperation.en-us.lg.lg

```diff
@@ -8,15 +8,23 @@
 
 ${'#'} displayOperation(op, property, entity)
 - IF: \${op && property && entity.value} 
-        - \${operationName(op)} "\${entity.text}" to \${propertyName(property)}
+        - \${operationName(op)} \${template(`\${property}Entity_Value`, formatValue(entity.value))} to \${propertyName(property)}
```

Comment:
>`\${property}Entity_Value [](start = 44, length = 25)

This will not work--you need to figure out the entity from the entity information which might be property named  off the property but might be a prebuilt entity.